### ISSUE TITLE
ICML-2019-MASS: Masked Sequence to Sequence Pre-training for Language Generation

### ISSUE THREAD

**Issue by BrambleXu** [BrambleXu/knowledge-graph-learning#156]
arXiv-2019/03-SCIBERT: Fine-tune BERT for Extractive Summarization

**一句话总结：**

**资源：**

- [pdf]()
- [code](
- [paper-with-code](

**关键字：**

- dataset: 
- 

**笔记：**

**模型图：**

**结果**：

**接下来要看的论文**：
